.github/workflows/ghpages-deploy.yml

```diff
@@ -1,37 +1,35 @@
 name: deploy-docusaurus
 
 on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+    push:
+        branches: [master]
 
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+    # Allows you to run this workflow manually from the Actions tab
+    workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  publish:
-    runs-on: ubuntu-latest
-    steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - name: Check out repo
-        uses: actions/checkout@v2
-      # Node is required for npm
-      - name: Set up Node
-        uses: actions/setup-node@v2
-        with:
-          node-version: "14"
-      # Install and build Docusaurus website
-      - name: Build Docusaurus website
-        run: |
-          npm install 
-          npm run build
-      - name: Deploy to GitHub Pages
-        if: success()
-        uses: crazy-max/ghaction-github-pages@v2
-        with:
-          target_branch: gh-pages
-          build_dir: build
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+            # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+            - name: Check out repo
+              uses: actions/checkout@v2
+            # Node is required for npm
+            - name: Set up Node
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "14"
+            # Install and build Docusaurus website
+            - name: Build Docusaurus website
+              run: |
+                  npm install 
```

Comment:
This could lead us into issues. I would prefer if we did `yarn install`

Yarn isnt installed by default so installing yarn is important

Reply:
I haven't tested it with yarn but I'm assuming the only difference would be I need to add `npm install -g yarn` and then change all the npm commands to `yarn`?

Reply:
yep. We do want to stick with the `yarn.lock` file and thats why using npm is dangerous